rllib/algorithms/es/es.py

```diff
@@ -357,6 +357,10 @@ def do_rollouts(self, params, timestep_limit=None):
             eval_lengths=eval_lengths,
         )
 
+    def stop(self):
```

Comment:
this function needs to be added to es to meet the Worker API. If it isn't added then there is an error that occurs during the test that isn't caught since it is inside of the worker manager which doesn't raise the error.